components/Searchbar/Searchbar.styled.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import Svg from "../Svg";
+
+const StyledForm = styled.form`
+  margin: 25px 0 0;
+  padding: 0 25px;
+  position: relative;
+  display: flex;
+  justify-content: center;
+`;
+
+const StyledInput = styled.input`
```

Comment:
here you use relative values: rem and vw, maybe it would be nice to stay consistent with px or rem/em. But could be something for the future